FreeAgent.postman_collection.json

```diff
@@ -3032,6 +3032,481 @@
 			],
 			"protocolProfileBehavior": {}
 		},
+		{
+			"name": "Credit Notes",
```

Comment:
I wonder if it might be more intuitive for the user to put these between Contacts and Email addresses as at present all the endpoints in the collection are arranged in an alphabetical order -- I just imported the collection into my Postman and it took me a moment to find Credit Notes!